x-pack/test/functional/services/ml/index.ts

```diff
@@ -202,6 +204,7 @@ export function MachineLearningProvider(context: FtrProviderContext) {
     jobSelection,
     jobSourceSelection,
     jobTable,
+    expandedJobDetails,
```

Comment:
Super duper nit: we should keep this ordered alphabetically so move this up after `dataVisualizerTable`, or a better approach (to me) is to rename 'expandedJobDetails' to something like `jobExpandedDetails` to match with the rest.